src/components/PrimaryPanes/Sources.css

```diff
@@ -79,6 +79,7 @@
 
 .sources-panel .source-footer {
```

Comment:
I think this should be renamed because it has nothing to do with footer
I'd also like to move this into a new PrimaryPane.css file as this is not to related to Sources component at all.
Any thoughts about this?

Reply:
agreed.